modules/translation/translation.go

```diff
@@ -26,6 +26,7 @@ type Locale interface {
 // LangType represents a lang type
 type LangType struct {
 	Lang, Name string // these fields are used directly in templates: {{range .AllLangs}}{{.Lang}}{{.Name}}{{end}}
+	Offset     int
```

Comment:
I think that Offset should definitely be documented, as the mechanism now needs a bit of explanation.